### PR TITLE
Sqreen integration

### PR DESCRIPTION
**Pull request by arnaudbreton**
### What does this PR do?

Describes the Sqreen integration

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: sqreen/manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "integration_id": "sqreen",
+    "name": "sqreen",
+    "manifest_version": "0.0.1",
+    "guid": "318a1d52-85ec-47ef-8174-72017881b360",
+    "maintainer": "support@sqreen.com",
```
Comment (anshumgargdd) [inline sqreen/manifest.json:6]: let's update this to help@datadoghq.com

File: sqreen/manifest.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "type": "crawler",
+    "support": "contrib",
+    "supported_os": [
+      "linux",
+      "mac_os",
+      "windows"
+    ],
+    "is_public": false,
+    "categories": [
+      "security"
```
Comment (anshumgargdd) [inline sqreen/manifest.json]: Can we also add "log collection" in categories

File: sqreen/manifest.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+    "integration_id": "sqreen",
+    "name": "sqreen",
+    "manifest_version": "0.0.1",
```
Comment (anshumgargdd) [inline sqreen/manifest.json]: please update the version to "1.0.0". Just following convention and it might fail our CI checks.

**Review by anshumgargdd — CHANGES_REQUESTED** [review]
Just a couple of changes that I requested on manifest. If you are happy with the dashboard feel free to make a PR to integrations-extras repos and I can have the agent intg team take a look at it,

File: sqreen/manifest.json (hunk 1 above)
Comment (arnaudbreton) [reply sqreen/manifest.json:6]: @anshumgargdd how will questions sent to this email be redirected to our team?

File: sqreen/manifest.json (hunk 3 above)
Comment (arnaudbreton) [reply sqreen/manifest.json]: @anshumgargdd for future releases, will we bump it too? Or it'll stay "1.0.0" forever

**Comment by arnaudbreton** [comment]
> Just a couple of changes that I requested on manifest. If you are happy with the dashboard feel free to make a PR to integrations-extras repos and I can have the agent intg team take a look at it,

Thanks for the feedback @anshumgargdd. 

Did you get a chance to review the assets too? I wasn't sure they're defined as expected.

**Pull request closed without merging**